scripts/docker_entrypoint.sh

```diff
@@ -67,7 +67,7 @@ function start_machine {
     then
         # you should provide the config file path and host settings yourself in case you want to use custom options!
         echo "Using default arguments."
-        "$BINARY_PATH"/sc-machine -c "$CONFIG_PATH" -e "$BINARY_PATH/extensions"
+        "$BINARY_PATH"/sc-machine -c "$CONFIG_PATH" -e "$EXTENSIONS_PATH"
```

Comment:
I couldn't find which line assigns value to EXTENSIONS_PATH

Reply:
docker-compose.yml

Reply:
ok, add it to usage, now usage mentions all env vars except for EXTENSIONS_PATH